envoy/config/subscription.h

```diff
@@ -250,5 +251,7 @@ struct SubscriptionStats {
                          GENERATE_TEXT_READOUT_STRUCT, GENERATE_HISTOGRAM_STRUCT)
 };
 
+constexpr uint32_t PERIODIC_STATS_TIMER_REFRESH_MS = 10 * 1000;
```

Comment:
resolution of this metric is not critical, so I picked an timer refresh rate here that I imagined to be not intrusive, but thinking about it making this something like every second should also not really have any impact. I'll wait for the reviewer to comment before changing anything.